jazz-ui/src/app/environment-module/environment.module.ts

```diff
@@ -0,0 +1,53 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {EnvLogsSectionComponent} from '../pages/environment-logs/env-logs-section.component';
```

Comment:
Please be consistent with indentation (lines 1 and 2 vs others)

Reply:
did the changes